install_config/cluster_metrics.adoc

```diff
@@ -168,15 +168,15 @@ $ oc process -f metrics-deployer.yaml -v \
 
 [WARNING]
 ====
-The size requirement of the Cassandra storage is dependent on the cluster size. It is
-the admin's responsibility to make sure that the size requires are sufficent for their setup and
-to monitor usage to make sure that the disk does not become full.
+The size requirement of the Cassandra storage is dependent on the cluster size.
+It is the administrator's responsibility to ensure that the size requirements
+are sufficient for their setup and to monitor usage to ensure that the disk does
+not become full.
```

Comment:
@ahardin-rh Suggest treating this like the WARNING admonition later, with just the "Data loss..." sentence being in the box.